src/main/java/com/hydratereminder/command/chat/ChatMessageSender.java

```diff
@@ -0,0 +1,73 @@
+package com.hydratereminder.command.chat;
```

Comment:
Would be great to have those classes under `com.hydratereminder.chat` instead of `com.hydratereminder.command.chat` :)